lib/AppInfo/Application.php

```diff
@@ -100,21 +102,28 @@ public function register(IRegistrationContext $context): void {
 		$cacheListener = $this->getContainer()->get(CacheListener::class);
 		$cacheListener->listen();
 
-		if (\interface_exists(IAPIWidgetV2::class)) {
+		if (interface_exists(IAPIWidgetV2::class)) {
 			$context->registerDashboardWidget(RecentPagesWidget::class);
 		}
 
 		if (method_exists($context, 'registerTeamResourceProvider')) {
 			$context->registerTeamResourceProvider(CollectiveTeamResourceProvider::class);
 		}
+
+		if (interface_exists('OCP\SetupCheck\ISetupCheck') && method_exists($context, 'registerSetupCheck')) {
```

Comment:
```suggestion
		if (interface_exists(\OCP\SetupCheck\ISetupCheck::class) && method_exists($context, 'registerSetupCheck')) {
```

We can use the class magic constant here